NorthstarDedicatedTest/serverauthentication.cpp

```diff
@@ -532,10 +559,26 @@ void CServerGameDLL__OnRecievedSayTextMessageHook(void* self, unsigned int sende
 
 	g_ServerAuthenticationManager->m_additionalPlayerData[sender].sayTextLimitCount++;
 
-	// todo: could censor messages here if we have a banned word list, we do not currently have one of these
-	// could possibly make this call a script codecallback, or smth
+	bool shouldDoChathooks = strstr(GetCommandLineA(), "-enableChatHooks");
```

Comment:
cmdline arguments are lowercase, i think `-enablechathooks` or something like that makes more sense